test/integration/providers/docker.go

```diff
@@ -5,12 +5,16 @@ package providers
 
 import (
 	"context"
+	"fmt"
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/client"
 	"io"
+	"io/ioutil"
+	"os/exec"
+	"strings"
 )
 
-func RunDockerCommand(container string, cmd string) {
+func RunDockerCommandAttach(container string, cmd string) {
```

Comment:
Could you add a short comment on what those two functions do, and how they differ?